src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector128.cs

```diff
@@ -4328,12 +4338,17 @@ internal static Vector128<byte> SubtractSaturate(Vector128<byte> left, Vector128
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         [CompExactlyDependsOn(typeof(AdvSimd.Arm64))]
         [CompExactlyDependsOn(typeof(Sse2))]
+        [CompExactlyDependsOn(typeof(PackedSimd))]
```

Comment:
Do we know why these methods have CompExactlyDependsOn applied? It looks like they obey the rules necessary to not have it.

Reply:
https://github.com/dotnet/runtime/issues/82559